lib/main.js

```diff
@@ -0,0 +1,76 @@
+// import this polyfill in order to make webapp compatible with IE 11
```

Comment:
I'm curious whether this and `app.js` could be merged into a single file?

Reply:
I think it could, but didn't want to create any more vectors for possible regressions!